githubscript/docker.d/worker.yml

```diff
@@ -20,4 +20,10 @@ github_projects:
         github_token: dummy
         github_owner: non-existing-owner
         github_repo_name: non-existing-repo
+    'COT_PRODUCT == "mobile" && ENV == "dev"':
+      mock:
+        allowed_actions: ["release"]
+        github_token: dummy
+        github_owner: non-existing-owner
```

Comment:
Same here, we could wire this to your fork, so we sure this works end-to-end. The `mock` product is quite simple doesn't do much.